dali/test/python/jax/test_integration_multigpu.py

```diff
@@ -125,3 +133,58 @@ def test_dali_sequential_sharded_tensors_to_jax_sharded_array_manuall():
         # Assert correct backing devices for shards
         assert jax_array.device_buffers[0].device() == jax_shard_0.device()
         assert jax_array.device_buffers[1].device() == jax_shard_1.device()
+
+
+def test_dali_sequential_sharded_tensors_to_jax_sharded_array_iterator_multiple_outputs():
+    assert jax.device_count() > 1, "Multigpu test requires more than one GPU"
```

Comment:
Assert of just SkipTest?

Reply:
I wanted to make sure that this test will never happen on a machine with one GPU. So with how it is now we are sure that this test actually test multi GPU integration.
Skipping is silent so it may happen that if something breaks in the CI we will stop testing multi GPU and will not find out about it. This is something I would like to avoid.

Reply:
Understood